numpy/lib/npyio.py

```diff
@@ -1642,9 +1716,11 @@ def genfromtxt(fname, dtype=float, comments='#', delimiter=None,
     # Process the missing values ...............................
     # Rename missing_values for convenience
     user_missing_values = missing_values or ()
+    if isinstance(user_missing_values, bytes):
+        user_missing_values = user_missing_values.decode('latin1')
```

Comment:
Can this happen?  If so, use `asstr` instead of explicit decoding

Reply:
`asstr` does seem better.

Reply:
Chnged my mind.